markdown/usage/data_guidelines.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Resource Estimation and Data sharing
+subtitle:
```

Comment:
Needs a subtitle 👍

Reply:
Maybe this, but the other way around? The current title is too long really, as it is:

![image](https://user-images.githubusercontent.com/465550/80714248-ed882f80-8af4-11ea-8569-a3477c517bfc.png)

If reversed then it looks better:

![image](https://user-images.githubusercontent.com/465550/80714278-fda00f00-8af4-11ea-8aee-570c4911be2d.png)

Reply:
Maybe something like this instead?

![image](https://user-images.githubusercontent.com/465550/80714362-20cabe80-8af5-11ea-972c-f44019d364eb.png)

```suggestion
subtitle: How to plan your project with resource estimation and data sharing guidelines.
```

Reply:
I like your suggestion better. My only worry is finding it from the front page.

Reply:
I really can't think of anything short though which is better.

Reply:
I think this last suggestion looks good. If this makes the information visible enough for the users or not I can't really tell